spec/vc.md

```diff
@@ -0,0 +1,80 @@
+
+This optionality can be difficult to implement consistently across languages. As a consequence, this specification defines a strict subset of the VC Data Model v1.1 that supports the [plain JSON syntax](https://www.w3.org/TR/vc-data-model/#json), represented by the following table. As a utility JSON Schemas for [Verifiable Credentials](vc-11.json) and [Verifiable Presentations](vp-11.json) are provided to aid in the validation of conformant documents.
+
+## Verifiable Credential Data Model
+
+Following from [this data model](https://www.w3.org/TR/vc-data-model/#basic-concepts).
+
+| Property      | JSON Representation | Required | Notes          |
+| ------------- | ------------------- | -------- | -------------- |
+| `@context`    | Array of strings    | Yes      | Contexts defining the meaning of terms within the credential. Must include at least `"https://www.w3.org/2018/credentials/v1"`. |
+| `id`          | String              | Yes      | A URI representing a unique identifier for the credential. Recommended to be of form `urn:uuid:3978344f-8596-4c3a-a978-8fcaba3903c5`. |
+| `type`        | Array of strings    | Yes      | Type(s) of the credential. Must include `VerifiableCredential`. |
+| `issuer`      | String              | Yes      | A DID representing a unique identifier for the entity that issued the credential. |
+| `issuanceDate`| String              | Yes      | [XML Datetime](https://www.w3.org/TR/xmlschema11-2/#dateTime) value for when the credential was issued. |
+| `expirationDate` | String           | No       | [XML Datetime](https://www.w3.org/TR/xmlschema11-2/#dateTime) value after which the credential is no longer valid. |
+| `credentialSubject` | Object        | Yes      | Data about the subject of the credential. Can be any JSON object. |
+| `credentialSubject.id` | Object     | Yes      | A DID representing a unique identifier for whom the credential's claims are made. |
+| `credentialStatus` | Object defined by [Credential Status](#credential-status) | No | Only to be used with [Status List 2021](https://www.w3.org/community/reports/credentials/CG-FINAL-vc-status-list-2021-20230102/). |
+| `credentialSchema` | Object defined by [Credential Schema](#credential-schema) | No | Recommended. Only to be used with the type [`JsonSchema`](https://w3c.github.io/vc-json-schema/#jsonschema). |
+| `evidence`    | Object              | No       | Any JSON object as per [Evidence](https://www.w3.org/TR/vc-data-model/#evidence). |
+
+**Additional Notes:**
+- The `credentialSubject` property can be any JSON object. It is recommended that this object is defined by an associated `credentialSchema`.
+- No [JSON-LD processing](https://www.w3.org/TR/vc-data-model/#json-ld) is performed.
+- Embedded proofs, using the `proof` property must not be present.
+- The `type` property may only contain `VerifiableCredential` or the URI of a JSON Schema, if one is used for the credential.
```

Comment:
This is not how we do it currently,

```
      const vc = await VerifiableCredential.create({
        type    : 'TBDeveloperCredential',
        subject : did.uri,
        issuer  : did.uri,
        data    : {
          legit: true
        }
      });
```

So from what you are saying our type "TBDeveloperCredential" cannot be added unless we also include the URI of the json schema is that correct?


If that is a hard rule then we will need to refactor our codebase

Reply:
yes, this isn't useful IMO and breaks the spec
if we have a need for a concrete type to switch on we should be using a JSON Schema